frontend/src/pages/vault/VaultLayout.jsx

```diff
@@ -1,11 +1,32 @@
 import React from 'react';
-import { NavLink, Outlet } from 'react-router-dom';
-import './vaultLayout.css';
+import { Outlet } from 'react-router-dom';
+ import './vaultLayout.css';
+import Sidebar from 'components/ui/Components/Sidebar/Sidebar';
 
 export function VaultLayout({ children }) {
+  const vaultItems = [
+    {
+      id: 'home',
+      name: 'Home',
+      link: '/',
+      icon: '•'
```

Comment:
<img width="372" alt="image" src="https://github.com/user-attachments/assets/419c609d-e1ab-4f8a-8a55-4ee2beb0d143" />

Also, title styles do not match Figma